Automattic/themes
# Update: Rename “_dsgnsystm“ to “Varia”

#### Changes proposed in this Pull Request:

This is a pretty big find-and-replace that renames the theme to “Varia”. For some reason the leading `_` in `_dsgnsystm` doesn’t agree with Trac. 

**Steps**

- [x] Rename `_dsgnsystm` to `varia` in [theme-dev-util](https://github.com/Automattic/theme-dev-utils) repo. [Done](https://github.com/Automattic/theme-dev-utils/pull/1).
- [x] Rename `_dsgnsystm` to `varia` in themes repo. e2d0179
- [x] Do a find and replace on `_dsgnsystm` and change it to `varia` on both the parent-theme and the build tool. b14a1fa
- [x] Do the same find and replace on `Redhill` theme. b22b157
- [x] Pass the two tests below.
- [x] Compile and merge this PR and [this one](https://github.com/Automattic/theme-dev-utils/pull/1) to their master branches.
- [x] Delete `_dsgnsystm` on wpcom and commit + deploy `varia` theme.

#### Varia Testing Instructions: 

1. `cd` to `/themes/varia`
2. Run `npm install`
3. Run `npm run build` and make sure it works without errors.
4. If it does, then run `npm run child-theme` 
5. Name your theme when the prompt asks.
6. `cd` to you `your-theme`
7. Run `npm run build` 

#### Redhill Testing Instructions: 

1. `cd` to `/themes/redhill`
2. Run `npm install`
3. Run `npm run build` and make sure it works without errors.
